packages/components/utils/src/MissingContent/MissingContent.tsx

```diff
@@ -0,0 +1,29 @@
+import { cx } from 'emotion';
+import React from 'react';
+import { getStyles } from './MissingContent.styles';
+import type { CommonProps } from '@contentful/f36-core';
+import { Text } from '@contentful/f36-typography';
+
+export interface MissingContentProps extends CommonProps {
+  title?: string;
+}
+
+export const MissingContent = ({
+  className,
+  testId = 'cf-ui-MissingContent',
+  title,
+  ...otherProps
+}: MissingContentProps) => {
+  const styles = getStyles();
+
+  return (
+    <Text
+      {...otherProps}
+      data-test-id={testId}
+      className={cx(styles.missingContent, className)}
+      title={title}
```

Comment:
In the cases i want to use the component it would be quite helpful to have further information e.g. required content is missing in the default language, description is not yet provided etc. 

So its low-key important for people that are wondering what this placeholder stands for. but not sure if it is crucial information.

Reply:
We could also enforce an `aria-label` prop.

Reply:
+1 for `aria-label` "forced"

Reply:
But aria-label will not appear on hover like title, or? Should we then have both?

Reply:
Both are good yes 👍🏼